lib/releaselib.sh

```diff
@@ -833,3 +846,51 @@ release::docker::release () {
     logrun $GCLOUD config set account $USER@google.com
   fi
 }
+
+###############################################################################
+# Determine whether the release was built with Bazel.
+# @param kube_root - Root of kubernetes tree
+# @return 0 if built with Bazel, 1 otherwise
+release::was_built_with_bazel() {
```

Comment:
Why not just check to see if [[ -d $kube_root/bazel-bin ]] ?  That should be indicator enough and it's a one liner.  Iyou wanted to be pedantic, maybe check that there are contents in bazel-bin (also a one liner):
[[ $(ls -A $kube_root/bazel-bin) ]]